lib/i18n-transform.js

```diff
@@ -28,41 +28,51 @@ var _getLanguageFromI18n = function (i18nObj) {
         return null;
     },
 
-    _isWildcard = function (language) {
+    _isWildcardMatch = function (language) {
         return language.code === "*";
     },
 
-    _isLanguageCodeMatch = function (i18nLanguage, language) {
-        return language.code.toUpperCase() === i18nLanguage.code.toUpperCase();
+    _isLanguageCodeMatch = function (languageA, languageB) {
+        return languageA.code.toUpperCase() === languageB.code.toUpperCase();
```

Comment:
At a minimum code will be always supplied in the case of an accept language. en = code en-GB = code + region.

Maybe we could have a check for double safety but I don't see it happening.